.github/workflows/lint.yml

```diff
@@ -10,24 +10,26 @@ jobs:
   build:
     name: lint
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ["3.8", "3.11"]
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Python ${{ matrix.python-version }}
+
+      - name: Set up Python
         uses: actions/setup-python@v3
         with:
-          python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
+          python-version: 3.10.11
```

Comment:
I would recommend pinning this to 3.10 instead of a specific subversion. Let the setup pick the latest.

Reply:
3.10 worked, I just had to put it in quotes! it defaults to latest (3.10.11 right now)
implemented in commit 322a43d382d0676c4d86f7ed536a2ef48c2321eb